src/test/java/redis/clients/jedis/tests/ProtocolTest.java

```diff
@@ -119,4 +120,17 @@ public void nullMultiBulkReply() {
     List<String> response = (List<String>) Protocol.read(new RedisInputStream(is));
     assertNull(response);
   }
-}
\ No newline at end of file
+
+  @Test
```

Comment:
You can use the `expected` annotation parameter to test for exceptions, no need to try-catch inside test.